send/send.c

```diff
@@ -2538,11 +2538,13 @@ int mutt_send_message(SendFlags flags, struct Email *e_templ, const char *tempfi
    * 3) we are resending a message
    * 4) we are recalling a postponed message (don't override the user's saved settings)
    * 5) we are in batch mode
+   * But 3, 4, and 5, can be overridden with '-C' in the command line (flags & SEND_CLI_CRYPTO)
    *
    * This is done after allowing the user to edit the message so that security
    * settings can be configured with send2-hook and $edit_headers.  */
   if ((WithCrypto != 0) && (e_templ->security == 0) &&
-      !(flags & (SEND_BATCH | SEND_POSTPONED | SEND_RESEND)))
+      ((flags & SEND_CLI_CRYPTO)
+       || !(flags & (SEND_BATCH | SEND_POSTPONED | SEND_RESEND))))
```

Comment:
I thought that, as you said (@flatcap), it could be Cli Crypto, and then it overrides all three modes, not just batch.  Does it sound good to you?

Reply:
Yes, it does :-)

Reply:
Tomorrow, I'll start trying to document it.

Have a lovely night! :-)